sdk/textanalytics/ai-text-analytics/src/textAnalyticsClient.ts

```diff
@@ -317,22 +315,23 @@ export class TextAnalyticsClient {
     }
 
     const authPolicy = isTokenCredential(credential)
-      ? bearerTokenAuthenticationPolicy(credential, DEFAULT_COGNITIVE_SCOPE)
-      : createTextAnalyticsAzureKeyCredentialPolicy(credential);
+      ? bearerTokenAuthenticationPolicy({ credential, scopes: DEFAULT_COGNITIVE_SCOPE })
+      : textAnalyticsAzureKeyCredentialPolicy(credential);
 
     const internalPipelineOptions: InternalPipelineOptions = {
       ...pipelineOptions,
       ...{
         loggingOptions: {
           logger: logger.info,
-          allowedHeaderNames: ["x-ms-correlation-request-id", "x-ms-request-id"]
+          additionalAllowedHeaderNames: ["x-ms-correlation-request-id", "x-ms-request-id"]
         }
       }
     };
 
-    const pipeline = createPipelineFromOptions(internalPipelineOptions, authPolicy);
+    const pipeline = createPipelineFromOptions(internalPipelineOptions);
```

Comment:
createPipelineFromOptions doesn't add in any serialization/deserialization policies. You want createClientPipeline from `core-client` instead

Reply:
Fixed.